ExampleMod/Items/Accessories/ExampleCloudinaBottle.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Items.Accessories
+{
+	public class ExampleCloudinaBottle : ModItem
+	{
+		public override string Texture => "Terraria/Item_" + ItemID.CloudinaBottle;
```

Comment:
At least make your own texture, this just makes the two items functionally and visually identical.

Reply:
Irrelevant.